src/main/java/org/spongepowered/api/entity/living/player/Player.java

```diff
@@ -68,7 +69,7 @@
  * <p>Any methods called on Player that are not on User do not store any data
  * that persists across server restarts.</p>
  */
-public interface Player extends Humanoid, User, Viewer, ChatTypeMessageReceiver, CommandSource {
```

Comment:
ChatTypeMessageReceiver is not a MessageReceiver?

Reply:
[Nope](https://github.com/SpongePowered/SpongeAPI/blob/api-8/src/main/java/org/spongepowered/api/text/channel/ChatTypeMessageReceiver.java#L40)

Reply:
I feel like we could extend MessageReciever and default the methods to call the chat type methods with ChatType.CHAT?

Reply:
I'll put that into another PR (or directly onto `api-8`)

Reply:
Do it directly in api-8.

Reply:
Okay.